src/server/server/dal.py

```diff
@@ -225,16 +230,16 @@ def get_user_actions_on_docs(users, from_datetime):
     return actions_by_user
 
 
```

Comment:
I think the fact that user action is stored as a string in the database (and the string instances) is an implementation detail of the database and so this method should remains private. And if the html handler uses also string representation as private format, it should have it's own strings and its own mapper.

Said another way : If we decide the shared representation is enum, the communication between two elements (here database and handlers) should be done by this representation.

The alternative is: If we concidere that the enum as shared representation creates too much boilerplate and is over-engineering, then we make the string format the shared format, and so it deletes all those mappers.

I thought this expansion joint (intermediate enum format) could be usefull rather than manipulate string directly (string are fragiles, migration, performance...) but I don't really mind the alternative if you prefere.